packages/react-button/src/components/Button/Button.tsx

```diff
@@ -1,20 +1,20 @@
 import * as React from 'react';
-import { useButton } from './useButton';
 import { renderButton } from './renderButton';
+import { useButton } from './useButton';
 import { useButtonStyles } from './useButtonStyles';
 import type { ButtonProps } from './Button.types';
 
 /**
  * Buttons give people a way to trigger an action.
  */
-export const Button: React.FunctionComponent<ButtonProps & React.RefAttributes<HTMLElement>> = React.forwardRef<
-  HTMLElement,
-  ButtonProps
->((props, ref) => {
+export const Button: React.ForwardRefExoticComponent<
+  ButtonProps & React.RefAttributes<HTMLButtonElement>
+> = React.forwardRef<HTMLButtonElement, ButtonProps>((props, ref) => {
```

Comment:
declaring ForwardRefExoticComponent is a bit tangled up in Ben's forwardRef PR and Elizabeth's follow-on PR.  @ecraig12345 - What should the declaration be if any?

Reply:
Let's keep that separate since the resolution is still not known on that one.